qemu/tests/numa_memdev_options.py

```diff
@@ -1,24 +1,29 @@
 import logging
+import re
 
 from avocado.utils import process
 
 from virttest import error_context
 from virttest import utils_misc
+from virttest import env_process
 from virttest.staging import utils_memory
 from virttest.compat_52lts import decode_to_text
+from virttest.utils_numeric import normalize_data_size
+from virttest.staging.utils_memory import get_huge_page_size
```

Comment:
utils_memory has been imported in L9, why need import get_huge_page_size respectively?